### ISSUE TITLE
piperider runs on wrong schema by default

### ISSUE THREAD

**Issue by VincentZhx** [InfuseAI/piperider#666]
**Describe the bug**
The command `piperider run --table <table_name>` points to the wrong schema

**To Reproduce**
1. Setup environment 
```
Core:
  - installed: 1.4.6
  - latest:    1.4.6 - Up to date!

Plugins:
  - postgres: 1.4.6 - Up to date!
  - redshift: 1.4.0 - Up to date!
```
2. Execute command 
```
dbt_core_test.sql: select 1, getdate()
$ dbt run --select dbt_core_test
$ piperider run --table dbt_core_test
```
```
DataSource: dev
─────────────────────────────────────────────────────────────────────────────────────────────────────── Validating ───────────────────────────────────────────────────────────────────────────────────────────────────────
everything is OK.
─────────────────────────────────────────────────────────────────────────────────────────────────────── Profiling ────────────────────────────────────────────────────────────────────────────────────────────────────────
[0/1] METADATA      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0/? 0:00:00Error: No such table 'vincent_dbt_dev.dbt_core_test'
```

**Expected behavior**
Piperider should get the correct table from schema set in the dbt profiles.yml

**Screenshots**
```
15:03:18  Running with dbt=1.4.6
15:03:19  Found 107 models, 4 tests, 1 snapshot, 0 analyses, 699 macros, 1 operation, 0 seed files, 61 sources, 0 exposures, 0 metrics
15:03:19  
15:03:19  Concurrency: 4 threads (target='dev')
15:03:19  
15:03:19  1 of 1 START sql view model dbt_vincentzhx.dbt_core_test ....................... [RUN]
15:03:19  1 of 1 OK created sql view model dbt_vincentzhx.dbt_core_test .................. [CREATE VIEW in 0.31s]
15:03:19  
15:03:19  Running 1 on-run-end hook
15:03:19  1 of 1 START hook: re_data.on-run-end.0 ........................................ [RUN]
15:03:19  1 of 1 OK hook: re_data.on-run-end.0 ........................................... [OK in 0.00s]
15:03:19  
15:03:19  
15:03:19  Finished running 1 view model, 1 hook in 0 hours 0 minutes and 0.87 seconds (0.87s).
15:03:20  
15:03:20  Completed successfully
15:03:20  
15:03:20  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1

DataSource: dev
─────────────────────────────────────────────────────────────────────────────────────────────────────── Validating ───────────────────────────────────────────────────────────────────────────────────────────────────────
everything is OK.
─────────────────────────────────────────────────────────────────────────────────────────────────────── Profiling ────────────────────────────────────────────────────────────────────────────────────────────────────────
[0/1] METADATA      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0/? 0:00:00Error: No such table 'vincent_dbt_dev.dbt_core_test'
```

**Desktop (please complete the following information):**
 - OS: linux
 - Python Version 3.9.4
 - DBT Version 1.4.6

**Additional context**

**Comment by DaveFlynn** [comment]
Hi @VincentZhx ,

From your CLI output we can see that dbt used `dbt_vincentzhx` while PipeRider detected `vincent_dbt_dev`.

Are you using any special configuration to define the schema in your dbt project files?

Could you please try this command to see if that completes without error:

```
piperider run --table dbt_vincentzhx.dbt_core_test
```

Thanks,
Dave

**Comment by DaveFlynn** [comment]
Thanks for the info @VincentZhx 

We are having some difficulty reproducing the issue.

Could you please let us know **where these two schemas are specified**: `vincent_dbt_dev` and `dbt_vincentzhx`?
E.g. `profiles.yml` in project directory, `~/.dbt`, set by ENV var etc. as per the [dbt docs](https://docs.getdbt.com/docs/core/connection-profiles#advanced-customizing-a-profile-directory).

Thanks

**Comment by kentwelcome** [comment]
This issue can be reproduced under the following conditions.
- The PipeRider project is inherited by dbt project
- The schema in the dbt profile is not the default data source. (ex. public, etc...)
- Run PipeRider with `--table` option to specify a table without giving schema

The root cause of this issue is PipeRider doesn't read the schema field of dbt profile when using `--table` to assign a specific table. 
https://github.com/InfuseAI/piperider/blob/04f4e0d3622f71f0061c4e5e527e913844779097/piperider_cli/runner.py#L728
This issue could be triggered in all kinds of data sources, not just AWS Redshift.

**Comment by kentwelcome** [comment]
This issue will be fixed in PipeRider v0.24.1
